packages/zora/src/Zora.ts

```diff
@@ -179,10 +174,39 @@ export const mint = async (
       $or: [ERC721_FILTER_ABSTRACT, ERC1155_FILTER_ABSTRACT],
     },
   }
-  return compressJson({
+  return {
     chainId,
     to: getExitAddresses(chainId, mintContracts), // We need this top level so the backend knows what contracts this applies to
     $or: [DIRECT_MINT_FILTER, UNIVERSAL_MINT_FILTER],
+  }
+}
+
+const v2MintFilter = (mint: MintActionParams) => {
+  const { chainId, contractAddress, tokenId, amount, recipient, referral } =
+    mint
+
+  return {
+    chainId,
+    to: getExitAddresses(chainId, ZORA_TIMED_SALE_STRATEGY),
```

Comment:
It does look like it's still possible to go through `0x777777722D078c97c6ad07d9f36801e653E356Ae` in addition to `0x777777722D078c97c6ad07d9f36801e653E356Ae` and the actual collection.

Reply:
Hmmmmmmmm - it looks like this address is `ZoraTimedSaleStrategy` on OP and `ZoraMintsManager` on Zora?